src/config.rs

```diff
@@ -831,6 +831,10 @@ impl TiKvConfig {
             }
             self.raft_store.region_split_size = default_raft_store.region_split_size;
         }
+
+        let delay_secs = self.raft_store.clean_stale_peer_delay.as_secs()
```

Comment:
How about `clean-stale-peer-delay = "0s"`? In that situation we should turn off DeleteFilesInRange.

Reply:
It's handled by `cleanup_overlap_ranges`.